superset-frontend/src/components/InfoTooltip/index.tsx

```diff
@@ -83,7 +83,11 @@ export default function InfoTooltip({
       overlayStyle={overlayStyle}
       color={bgColor}
     >
-      <Icons.InfoSolidSmall className="info-solid-small" viewBox={viewBox} />
+      <Icons.InfoSolidSmall
+        className="info-solid-small"
+        viewBox={viewBox}
+        iconSize="m"
```

Comment:
This part seems risky... I'm not sure how many places in the app this would affect. I suspect a lot. @geido do you remember our little alignment tricks here?

Reply:
At a minimum, we could set up the InfoTooltip to take an optional iconSize parameter and take it in here. Then, in the other file, instead of `<InfoTooltip tooltip={toolTipDescription} />` you could use `<InfoTooltip tooltip={toolTipDescription} iconSize="m" />`. Then it would only affect the one instance instead of all InfoTooltips in the app.

I swear we had another trick to vertically align these icons, though, besides making them teensy.

Reply:
You can avoid altering the InfoTooltip alltogether, and play with the viewbox attribute... it seems like `<InfoTooltip tooltip={toolTipDescription} viewBox='0 -7 28 28'/>` works OK. It's kind of an awkward-feeling attribute to fiddle with, though.

Reply:
I've pushed this viewBox change. But I'm curious to learn how it works. I tried fiddling with x, y, width, height values but I wasn't able to understand what was going on.